src/main/scala/com/webtrends/harness/component/akkahttp/routes/RouteGenerator.scala

```diff
@@ -93,6 +96,7 @@ object RouteGenerator {
                       mapRouteResult {
                         case Complete(response) =>
                           accessLogIdGetter.foreach(g => AccessLog.logAccess(reqWrapper, g(reqWrapper), response.status))
+                          finishTimer(response.status.intValue, timer)

```

Comment:
Let's have finishTimer take in a TimerStopWatch instead of an Option, and do a foreach over the timer option when we call it.

Reply:
Updated.